### PR TITLE
Optimize dashboard system usage tutorial

### PR DESCRIPTION
**Pull request by luoyuctl**
- Expansion of built-in dashboard system usage tutorial.
- Add more instructions on how to use custom functions to distinguish the types of custom functions.

### CODE REVIEW

File: docs/1.0/manual/microservice/dashboard-advanced.md [hunk 1]
```diff
@@ -59,36 +61,43 @@ LIMIT 100;
...
+### 关系运算函数
 * eq(value1, value2), 等价于 value1 == value2，返回boolean
+* neq(value1, value2), 等价于 value1 != value2，返回boolean
 * gt(value1, value2), 等价于 value1 > value2，返回boolean
 * gte(value1, value2), 等价于 value1 >= value2，返回boolean
 * lt(value1, value2), 等价于 value1 < value2，返回boolean
 * lte(value1, value2), 等价于 value1 <= value2，返回boolean
+
+### 条件运算函数
+* if(cond, true_expression, false_exprission), 等价于 cond ? true_expression : false_exprission
```
Comment (daskyrk) [inline docs/1.0/manual/microservice/dashboard-advanced.md:90]: typo `false_exprission `

File: docs/1.0/manual/microservice/dashboard.md [hunk 2]
```diff
@@ -253,19 +286,22 @@ LIMIT 100;
...
-* 不等于，!=<br>
-* 正则表达式，=~<br>
-* 数值类型的字段支持：<br>
-* 大于，><br>
-* 大于等于，>=<br>
-* 小于，<<br>
-* 小于等于，<=<br>
+字符串类型的字段支持：
+* 等于，=
+* 不等于，!=<
```
Comment (daskyrk) [inline docs/1.0/manual/microservice/dashboard.md:291]: redundant `<`

File: docs/1.0/manual/microservice/dashboard-advanced.md (hunk 1 above)
Comment (luoyuctl) [reply docs/1.0/manual/microservice/dashboard-advanced.md:90]: > typo `false_exprission `

if condition was false , case the false exprission

File: docs/1.0/manual/microservice/dashboard.md (hunk 2 above)
Comment (luoyuctl) [reply docs/1.0/manual/microservice/dashboard.md]: > redundant `<`

already changed

File: docs/1.0/manual/microservice/dashboard-advanced.md (hunk 1 above)
Comment (daskyrk) [reply docs/1.0/manual/microservice/dashboard-advanced.md:90]: is `expression`, not `exprission`

**Review by sfwn — APPROVED** [review]

**Pull request merged**